SoraFoundation/Classes/ViewModel/Preprocessors/InputViewModelPreprocessor.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+public protocol InputViewModelPreprocessing {
+    func preprocess(text: String) -> String
```

Comment:
Why `preprocess` when it's processing text actually?)

Reply:
Changed to ```process```